.changeset/nervous-nails-notice.md

```diff
@@ -0,0 +1,23 @@
+---
+"@latticexyz/world": minor
```

Comment:
I would opt for not exporting this from `world/external` yet to test it a bit and bundle the external feature release with a couple other libraries for the wallet connection flow. That means i would declare this as patch for now and add a note to the changeset that it's not ready for "stable" consumption yet.

Reply:
makes sense!